### PR TITLE
IS-13 Save user form does not work

### PR DESCRIPTION
**Pull request by jcbbb**

### CODE REVIEW

File: pkg/infrastructure/persistence/mappers.go [hunk 1]
```diff
@@ -35,7 +35,8 @@ func toDomainUser(dbUser *models.User) *user.User {
 		MiddleName: middleName,
 		Email:      dbUser.Email,
 		Password:   password,
-		//AvatarID:   dbUser.AvatarID,
+		Avatar:     toDomainUpload(dbUser.Avatar),
```
Comment (diyor28) [inline pkg/infrastructure/persistence/mappers.go]: I would rather move the nil check here. toDomainUpload should seg fault when handling a nil pointer

File: pkg/presentation/mappers/mappers.go [hunk 2]
```diff
@@ -102,6 +108,9 @@ func EmployeeToViewModel(entity *employee.Employee) *viewmodels.Employee {
 }
 
 func UploadToViewModel(entity *upload.Upload) *viewmodels.Upload {
+	if entity == nil {
```
Comment (diyor28) [inline pkg/presentation/mappers/mappers.go]: Here as well, I think it's better to keep these functions seg faulting when handling nil pointers

**Review by diyor28 — CHANGES_REQUESTED** [review]

File: pkg/presentation/mappers/mappers.go (hunk 2 above)
Comment (jcbbb) [reply pkg/presentation/mappers/mappers.go]: Done

File: pkg/infrastructure/persistence/mappers.go (hunk 1 above)
Comment (jcbbb) [reply pkg/infrastructure/persistence/mappers.go]: Done

**Pull request merged**